content/guides/tooling/typescript-support.md

```diff
@@ -95,9 +95,8 @@ If your specs files are in TypeScript, you should include the TypeScript definit
 
 Even if your project is JavaScript only, the JavaScript specs can know about the new command by referencing the file using the special triple slash `reference path` comment.
 
-```javascript
+```typescript
```

Comment:
This area appears to be addressing JS specifically, which is why the example had the triple slash references, but I do think this is likely confusing people. I would remove this paragraph "Even if your project is Javascript only....". The doc is about TypeScript after all.